src/trinary_sse42.h

```diff
@@ -247,4 +247,201 @@ static inline Trobject_t *trytes_from_trits_sse42(Trobject_t *trits)
     return trytes;
 }
 
+static inline Trobject_t *trits_from_trytes_sse42(Trobject_t *trytes)
+{
+    Trobject_t *trits = NULL;
+    int8_t *src = (int8_t *) malloc(trytes->len * 3);
```

Comment:
It might benefit from memory pool. Let's rework later.

Reply:
Got it.